Spring2021/ProjectIdeas_Rory-Covenant.md

```diff
@@ -0,0 +1,10 @@
+Rory's ideas:
+1.  browser extension that filters out spoiler search results.
+2.  Online game (something like "heads up" or "out of the loop")
+	- https://apps.apple.com/us/app/heads-up/id623592465
+	- https://play.google.com/store/apps/details?id=com.tastyrook.loop&hl=en_US&gl=US
+
+Covenant's Ideas:
+1. App that forces you to break down a project step by step in order to help manage its completion
+2. An application that randomly matches you with a a volunteering event happenning in your community. Maybe 'party feature' to ensure being matched at same event with friends/family
+3. Quacs Feature: Enter what requirements one still needs to fulfill to graduate and Quacs uses that to recommend classes for the following semester. Recommendations can further be filtered with 'preferences'
```

Comment:
We are actually already working on this! 
Repo: https://github.com/quacs/degree-planner
Associated data: https://github.com/quacs/quacs-data/tree/master/degree_requirements

Reply:
Wow! Really cool to find out that this is possible and in the works!